.github/workflows/PR.yml

```diff
@@ -70,7 +70,9 @@ jobs:
 
       - name: Run windows tests
         if: matrix.os == 'windows-latest'
-        run: ./gradlew mingwTest sqldelight-idea-plugin:check --stacktrace
+        run: |
+          ./gradlew mingwX64Test sqldelight-idea-plugin:check --stacktrace
+          ./gradlew mingwX86Test sqldelight-idea-plugin:check --stacktrace
```

Comment:
This runs the plugin check twice. We can just put the two test tasks into a single gradle run plus the plugin check.

Reply:
Fixed, thanks!